### PR TITLE
Fixed elasticsearch exclusion from JDL

### PR DESCRIPTION
**Pull request by Tcharl**
Fix the except keyword of elasticsearch to exclude repository generation in case of except.

Must be merged after https://github.com/jhipster/jhipster-core/pull/218 is.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: test/templates/import-jdl/search.jdl [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+	id Long
+	name String
+}
+
+entity WithoutSearch {
+	id Long
+	name String
+}
+
+search WithSearch with elasticsearch except WithoutSearch
```
Comment (MathieuAA) [inline test/templates/import-jdl/search.jdl]: Or just `search * with elasticsearch except WithoutSearch`

**Comment by Tcharl** [comment]
Tests will be green after the new release of jhipster-core including PR #218

**Comment by jdubois** [comment]
OK @Tcharl let's merge and see if the build passes indeed :-)

**Comment by MathieuAA** [comment]
v3.0.1 is out

**Comment by jdubois** [comment]
Sorry @MathieuAA do you mean we should switch to v3.0.1? I'm a bit lost here

**Comment by MathieuAA** [comment]
@jdubois I'll get a PR for v3.0.1 tonight and take of this.

**Comment by jdubois** [comment]
Oh sorry I'm just understanding my mistake

**Comment by MathieuAA** [comment]
Do you plan on releasing tonight?

**Comment by jdubois** [comment]
Yes thanks a lot @MathieuAA !!
I'm first planning to go to a great Korean restaurant for a friend's birthday, and then do a release if I don't come back home too late :-)

**Comment by MathieuAA** [comment]
@jdubois Ahah, bon appétit ! I'll get it done before you come back.

**Comment by jdubois** [comment]
Thanks @MathieuAA !! But that's a major release for you, are you confident it will be OK?

**Comment by MathieuAA** [comment]
What major release? It's already done for JCore (the only major release is for you I'm afraid ;)). My only issues for the past few days have been because of Yeoman!! I've lost hair because of this.

**Comment by DanielFran** [comment]
@jdubois, there is dependency with  https://github.com/jhipster/jhipster-core/pull/218 that is only available in jhipster-core v3.0.1.

**Pull request merged**